p2panda-stream/src/stream/ingest.rs

```diff
@@ -153,7 +153,7 @@ where
             // 2. Validate and check the log-integrity of the incoming operation. If it is valid it
             //    get's persisted and the log optionally pruned.
             let ingest_fut = async {
-                let log_id: L = header
```

Comment:
Is this a small clean up (because the compiler can derive that `log_id` is `L`) or is this doing something specific?

Reply:
Yeh, this is just a clean up :+1: